src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticateInfo.cs

```diff
@@ -9,21 +9,25 @@ namespace Microsoft.AspNetCore.Http.Authentication
     /// <summary>
     /// Used to store the results of an Authenticate call.
     /// </summary>
+    [Obsolete("See https://go.microsoft.com/fwlink/?linkid=845470")]
```

Comment:
This is the pattern we usually use for obsolete messages. https://github.com/aspnet/Home/wiki/Engineering-guidelines#breaking-changes cc @Eilon